dotworks/dotworks_server/forms.py

```diff
@@ -108,3 +108,20 @@ class InternshipCreationForm(forms.Form):
 
 class InscriptionAddForm(forms.Form):
     pass
+
+class ChangePasswordForm(forms.Form):
+    password = forms.CharField(
+        widget=forms.PasswordInput(),
+        label="Password antiga",
```

Comment:
We need to start using english. So it's best to already change labels to english